### PR TITLE
Add rails in rake task

### PR DESCRIPTION
**Pull request by emaglio**

### CODE REVIEW

File: tasks/assemble.rake [hunk 1]
```diff
@@ -61,6 +62,42 @@ namespace :future do
     end
   end
 
+  desc "TRB-rails: remove loader, rename render (render_v2_1) and run (run_v2_1)"
+  task :fix_trb_rails do
+    rails_path = File.join(root_path, "tmp/dist/lib/trailblazer/v2_1/rails/")
+
+    railtie_path = File.join(rails_path, "railtie.rb")
+    content = File.read(railtie_path)
+    content = content.gsub(/include Loader/, "")
```
Comment (emaglio) [inline tasks/assemble.rake]: Is there a better way to remove the line here instead to replace it with `""`

**Comment by seuros** [comment]
Rails part should be required manually.

**Comment by seuros** [comment]
Sorry, i didn't meant to close the PR. We can have the code but not the require.

File: tasks/assemble.rake [hunk 2]
```diff
@@ -61,6 +62,41 @@ namespace :future do
     end
   end
 
+  desc "TRB-rails: remove loader, rename render (render_v2_1) and run (run_v2_1)"
```
Comment (seuros) [inline tasks/assemble.rake]: I think we should avoid this hack.

**Pull request closed without merging**